lnc/log.go

```diff
@@ -0,0 +1,26 @@
+package lnc
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// Subsystem defines the sub system name of this package.
+const Subsystem = "LNCS"
```

Comment:
what does the `S` stand for out of curiosity? 🤓

Reply:
Subsystem? I just did not know any other way to make it 4 letters, happy to use 3 or use another naming.